content/en/observability_pipelines/setup/_index.md

```diff
@@ -36,6 +36,16 @@ There are several ways to get started with the Observability Pipelines Worker.
 
 This document walks you through the quickstart installation steps and then provides resources for next steps.
 
+## Deployment modes
```

Comment:
@barieom if you are good with the "Deployment modes" section up here and then "Updating deployment modes" later on, i can add this to the Datadog and Splunk guides as well.

Reply:
Barry is good with this and said only to add to Datadog setup.